doc/source/getting-started-linux-tutorial.rst

```diff
@@ -14,33 +14,35 @@ Volatility3 does not provide the ability to acquire memory.  Below are some exam
 Be aware that LiME raw format is not supported by volatility3, the padded or lime option should be used instead. `This issue contains further information <https://github.com/504ensicsLabs/LiME/issues/111>`_.
```

Comment:
This may not be necessary either, as the LiME project is no longer maintained.

Reply:
Thanks for pointing that out. I removed the LiME reference since the project is no longer maintained, and left a general note encouraging users to check compatibility and maintenance status when choosing memory acquisition tools.